packages/atomic/src/components/result-template-components/atomic-result-html/atomic-result-html.tsx

```diff
@@ -0,0 +1,52 @@
+import {Component, Prop, h, Element, State} from '@stencil/core';
+import {Result} from '@coveo/headless';
+import {ResultContext} from '../result-template-decorators';
+import {
+  Bindings,
+  InitializableComponent,
+  InitializeBindings,
+} from '../../../utils/initialization-utils';
+import {getStringValueFromResultORNull} from '../../../utils/result-utils';
+
+/**
+ * The `atomic-result-html` component renders the HTML value of a string result field.
+ *
+ * There is an inherent XSS security concern associated with the usage of this component.
+ *
+ * Use only with fields for which you are certain the data is harmless.
+ */
+@Component({
+  tag: 'atomic-result-html',
+  shadow: false,
+})
+export class AtomicResultHtml implements InitializableComponent {
```

Comment:
Why would a user use AtomicResultHTML instead of AtomicHTML if it's less safe?

Reply:
Both have the same level of "dangerousness". 

AtomicResultHTML uses AtomicHTML under the hood in its render method.

The difference is that AtomicResultHTML makes it easy to pull values from field results (so, data that is relevance in the context of trying to render result templates), as opposed to a generic "any string" property used by AtomicHTML.

Reply:
I see !